lightning-tag/data-collection.md

```diff
@@ -37,6 +37,7 @@ Lightning Tag will automatically execute any data collection rules that you have
   var namespace = window['lotame_' + lotameConfig.clientId] = {};
   namespace.config = lotameConfig;
   namespace.data = lotameTagInput.data || {};
+  namespace.cmd = namespace.cmd || [];
```

Comment:
There is going to be more than this, yes? This should now be part of our standard spec so that anyone implementing the tag now just adds push commands.

Reply:
Yes, there will be more. I left this as a draft PR but I wasn't sure what that meant. I was also using the github editor directly